.github/workflows/rosa-run-benchmark.yml

```diff
@@ -5,14 +5,38 @@ on:
     inputs:
       clusterName:
         description: 'Name of the cluster'
-        type: text
+        type: string
+      namespacePrefix:
```

Comment:
Do we have some use-case for setting `namespacePrefix` here? Thanks to 
```
concurrency: cluster_${{ inputs.clusterName || format('gh-{0}', github.repository_owner) }}
```
we are sure there is only one benchmark workflowrunning at each time. Is there some advantage of running each in a different namespace?

Reply:
Ideally we should avoid running multiple benchmarks against different namespaces in parallel.

Reply:
ok, I'll hardcode the prefix.